recipe/meta.yaml

```
@@ -32,29 +32,31 @@ requirements:
     - cachey >=0.2.1
     - certifi >=2020.6.20
     - dask >=2.15.0
```

This line is different in [setup.cfg](https://github.com/napari/napari/blob/main/setup.cfg#L45):
```suggestion
    - dask >=2.15.0,!=2.28.0
```

---

thanks!